public/swagger/Site.json

```diff
@@ -35,6 +35,9 @@
     "publicPreview": {
       "type": "boolean"
     },
+    "publishedAt": {
+      "type": "string"
```

Comment:
do we want to indicate this field uses a "date-time" format? not critical, but would make it easier to parse the results if you care.

Of course, if we're not following that format, then, no.